daffodil-test/src/test/resources/org/apache/daffodil/extensions/doubleFromRawLong/doubleFromRawLong.tdml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<tdml:testSuite xmlns:ex="http://example.com" xmlns="http://example.com"
+  xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:dfdlx="http://www.ogf.org/dfdl/dfdl-1.0/extensions"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"  >
+
+  <tdml:defineSchema name="doubleFromRawLong.dfdl.xsd">
+
+    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+    <dfdl:format ref="ex:GeneralFormat"
+      lengthUnits="bytes" lengthKind="explicit" length="8"
```

Comment:
In general, it's a bad idea to have the default value include an explicit length. Even though it works for this test, that rarely makes sense. Instead, I would recommend you remove `lengthKind="explicit"` and `length="8"` from this. The lengthKind value from GeneralFormat will be `implicit`, which is a sane default. And then you'll need to add `dfdl:lengthkind="explicit"` to the `LongValue` element. 

You also shouldn't need `dfdlx:emptyElementParsePolicy="treatAsEmpty"` for this tests. I would suggest removing that.

Same goes for your other tests.